deployment/jcdc.jsonnet

```diff
@@ -82,11 +94,12 @@
     apiVersion: 'apps/v1',
     kind: 'Deployment',
     metadata: {
+      namespace: 'jcdc',
+      name: 'jcdc' + $.config.nameSuffix,
       labels: {
         app: 'jcdc',
+        dev: $.config.dev,
       },
-      name: 'jcdc',
-      namespace: 'jcdc',
     },
     spec: {
       selector: {
```

Comment:
You also need to add dev to the label selector

Reply:
done.